README.md

```diff
@@ -8,17 +8,25 @@ PokeReader is a 3gx plugin that allows viewing information about 3ds Pokemon gam
 
 This information can be used to RNG shiny and high IV Pokemon, similar to lua scripts on emulators.
 
+## Commands
+- X+Y to Lock/Unlock
+- D-pad keys to navigate when unlocked (right & left to get in and out of the different tabs)
+- Start + Select to pause game and enable manual frame advancing
+- Select to advance Main RNG frames once
+- Start or A to unpause
```

Comment:
I almost wonder if `to` should be replaced with a semicolon in each of these lines:
```
- X + Y: Lock/Unlock
- Select: Advance one frame
```

Thoughts?